pkg/ca/ca.go

```diff
@@ -26,5 +27,5 @@ import (
 // fetching the CA trust bundle.
 type CertificateAuthority interface {
 	CreateCertificate(context.Context, identity.Principal, crypto.PublicKey) (*CodeSigningCertificate, error)
-	Root(ctx context.Context) ([]byte, error)
+	Root(ctx context.Context) ([]*x509.Certificate, error)
```

Comment:
To reflect the public interface, should we return a list of list of certificates?